docs/source/notebooks_and_ipython/kedro_and_notebooks.md

```diff
@@ -368,7 +422,7 @@ Similarly, the following creates a custom Jupyter kernel that automatically load
 kedro jupyter lab
 ```
 
-You can use any other Jupyter client to connect to a Kedro project kernel such as the [Qt Console](https://qtconsole.readthedocs.io/), which can be launched using the `kedro_iris` kernel as follows:
+You can use any other Jupyter client to connect to a Kedro project kernel such as the [Qt Console](https://qtconsole.readthedocs.io/), which can be launched using the `spaceflights` kernel as follows:
 
 ```bash
 jupyter qtconsole --kernel=kedro_iris
```

Comment:
``jupyter qtconsole --kernel=kedro_iris`` -> ``jupyter qtconsole --kernel=spaceflights``

Reply:
Thanks! I actually want to remove the section altogether, since I don't think this add much value. But to keep this PR simple, I will apply the changes.